src/models/alignment.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const Alignment = new Schema({
+  _id: {
+    type: String,
+    select: false,
+  },
+  desc: String,
```

Comment:
Not on you and not something you need to fix, but I wish we used `description` everywhere instead of `desc`.